deploy/006-omnichain-executorOwner-remote.ts

```diff
@@ -115,26 +106,37 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
     autoMine: true,
   });
-
   const omnichainExecutorOwner = await ethers.getContractAt(
     "OmnichainExecutorOwner",
     OmnichainExecutorOwner.address,
     ethers.provider.getSigner(deployer),
   );
+  const omnichainGovernanceExecutor = (await ethers.getContractAt(
+    "OmnichainGovernanceExecutor",
+    omnichainGovernanceExecutorAddress,
+    ethers.provider.getSigner(deployer),
+  )) as OmnichainGovernanceExecutor;
 
   if ((await omnichainGovernanceExecutor.owner()) === deployer) {
+    let tx = await omnichainGovernanceExecutor.addTimelocks([
+      normalTimelockAddress,
+      fastTrackTimelockAddress,
+      criticalTimelockAddress,
+    ]);
+    await tx.wait();
+
     const omnichainProposalSenderAddress = await getOmnichainProposalSender(networkName);
     await executeCommands(
       omnichainGovernanceExecutor,
       hre,
       deployer,
-      omnichainProposalSenderAddress,
-      10102,
+      omnichainProposalSenderAddress as any,
+      LZ_CHAINID["bsctestnet"],
```

Comment:
```suggestion
      LZ_CHAINID[hre.network.name],
```

Reply:
The source chain ID is now provided as a parameter instead of using hre.network.name. I've optimized this approach b15fc0d0147b04fc056215a5fb750c6dc4c7f544